routes/web.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-Route::middleware(['web', 'auth:' . config('admin-auth.defaults.guard'), 'admin'])->group(function () {
+Route::middleware(['auth:' . config('admin-auth.defaults.guard')])->group(function () {
```

Comment:
Now when `brackets/admin-auth` is not required, will `admin-auth.defaults.guard` work?

Reply:
Yes, it should work, because "dummy" config values are registered in service provider.